wercker.yml

```diff
@@ -25,18 +25,22 @@ build:
       name: go vet
       code: make govet
 
+    - script:
+      name: unit tests
+      code: make test
+
     - script:
       name: build
       code: make build
 
     - script:
-      name: unit tests
-      code: make test
-
+      name: manifests
+      code: make manifests
+      
```

Comment:
Dead space will result in confusing merge differences for editors which auto prune trailing whitespace